scripts/msr_export_attribute_mapper.py

```diff
@@ -0,0 +1,137 @@
+import csv
+import click
+from pathlib import Path
+
+autologic_member_attribute_keys = ['instructor', 'timing', 'grid', 'start', 'captain', 'gate']
+
+msr_to_autologic_member_attribute_map = {
+   "Timing & Scoring": 'timing',
+   "Grid": 'grid',
+   "Starter": 'start',
+   "Corner Captain": 'captain',
+   "Gate": 'gate'
+}
+
+def map_msr_export_work_assignments_to_autologic(name: str, assignments: list[str]):
+    """
+    Converts work assignments from an MSR export to the autologic member attribute names
+
+    Args:
+        name (str): The name of the registrant
+        assignments (list[str]): A list of work assignments they've elected in MSR
+    Returns:
+        dict[str, str]: The autologic member attributes with their name and empty strings for unelected attributes
+    """
+    return {
+        'name': name,
+        'instructor': '',
+        **{autologic_attribute_value: "TRUE" if msr_assignment_key in assignments else ""
```

Comment:
Is `Work Assignment` in the MSR export user-supplied? Does this mean that we're "qualifying" people if they indicate that they are interested in those roles when they register?

Reply:
Perhaps we can add something that first identifies the additions and then has the user approve or reject those additions? Or something to that effect.

I'm good either way for now, though, since you'll be the primary user of the script.

Reply:
Sounds good, feel free to merge whenever.